app/decorators/repository_decorator.rb

```diff
@@ -1,19 +1,37 @@
 # frozen_string_literal: true
 
 class RepositoryDecorator < ApplicationDecorator
+  include ActionView::Helpers::NumberHelper
+
   delegate_all
 
   def self.collection_decorator_class
     PaginatingDecorator
   end
 
   def languages
-    return '' unless language
+    return [] unless language
 
-    language.split(',').first(3).to_sentence
+    language.split(', ').first(3).map(&:strip)
   end
 
   def name
     link[/[\w_-]+$/]
   end
+
+  def issues_formatted
+    format_number(model.issues)
+  end
+
+  def stars_formatted
+    format_number(model.stars)
+  end
+
+  private
+
+  def format_number(number)
+    return (number || 0).to_s if (number || 0) <= 999
```

Comment:
Let me improve

Reply:
Question for you @adbatista . Should we be that defensive?

There's a check for `nil` in this method. Is it needed? In my humble opinion, If a client is passing `nil` for this method it's calling it incorrectly by definition.

Reply:
Yeah, I also don't think this method should have this concern

Reply:
I solved this concern about`nil`, just updating the `schema.rb`, adding default value as zero to issues and stars columns. I think this would be enough, what do you think?

Reply:
schema.rb is a auto generated file, yoou should not edit, change the migration to add a default value

Reply:
Sorry, I expressed myself wrong, I previously performed a rollback on the last migration I had created and created a new migration adding the new columns with default zero.